### PR TITLE
Integration of LDAP authentication.

### PR DESCRIPTION
**Pull request by sudhirparmar123**

### CODE REVIEW

File: app/services/dashboard.service.js [hunk 1]
```diff
@@ -10,7 +10,8 @@
 angular.module('resumeApp')
   .service('dashboardService', function ($http,$log) {
     // AngularJS will instantiate a singleton by calling "new" on this function
-var serverUrl = "http://172.26.35.51:8001/";
+    var serverUrl = "http://172.26.35.51:8001/";
+    //var serverUrl = "http://172.26.35.76:8001/";
```
Comment (csinghal) [inline app/services/dashboard.service.js]: Remove commented code

File: servers/routes/employee.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
```
Comment (csinghal) [inline servers/routes/employee.js]: Remove this code.

File: servers/routes/employee.js [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
 var express = require('express');
 var router = express.Router();
 var mongoose = require('mongoose');
 var Employee = require('../models/employee.js');
-var User = require('../models/user.js');
+//var User = require('../models/user.js');
```
Comment (csinghal) [inline servers/routes/employee.js]: Remove commented code

File: app/services/dashboard.service.js (hunk 1 above)
Comment (sudhirparmar123) [reply app/services/dashboard.service.js]: Fixed

File: servers/routes/employee.js (hunk 2 above)
Comment (sudhirparmar123) [reply servers/routes/employee.js]: Fixed

File: servers/routes/employee.js (hunk 3 above)
Comment (sudhirparmar123) [reply servers/routes/employee.js]: Fixed.

**Pull request merged**